imapserver/capability.go

```diff
@@ -75,6 +75,8 @@ func (c *Conn) availableCaps() []imap.Cap {
 				imap.CapMove,
 				imap.CapStatusSize,
 				imap.CapBinary,
+				imap.CapChildren,
+				imap.CapID,
```

Comment:
The ID extension is applicable to IMAP4rev2 as well, so should be moved below.

I've added a commit to better explain what these cap lists are: b33055e420b742b890c8a93e9a5b50163845bca3